deployments/boringssl-envoy-deployment.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: helloenvoy
+  labels:
+    app: boringssl-envoy
+spec:
+  type: NodePort
+  ports:
+    - port: 9000
+      targetPort: 9000
+      protocol: TCP
+      name: https
+  selector:
+    app: boringssl-envoy
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: boringssl-envoy-config
+data:
+  envoy-conf.yaml: |
+    static_resources:
+      listeners:
+      - address:
+          socket_address:
+            address: 0.0.0.0
+            port_value: 9000
+        filter_chains:
+          tls_context:
+            common_tls_context:
+              private_key_operations:
+                provider_name: qat
+                typed_config:
+                  "@type": "type.googleapis.com/qat.QatPrivateKeyOperationsConfig"
+                  section_name: ${QAT_SECTION_NAME}
+                  poll_delay: 2000000
```

Comment:
Maybe make `poll_delay` set by env variable too? Would be interesting to try different values in automatic perf testing.

Reply:
And maybe add a comment explaining that Kubernetes is not filling these values, but they are expected to be processed in the container.